Cargo.toml

```diff
@@ -26,21 +27,22 @@ travis-ci   = { repository = "inotify-rs/inotify" }
 
 [features]
 default = ["stream"]
-stream = ["futures", "mio", "tokio", "tokio-io", "tokio-reactor"]
+stream = ["futures-core-preview", "mio", "tokio-io", "tokio-net"]
 
 
 [dependencies]
-bitflags      = "1"
-futures       = { version = "0.1", optional = true }
-inotify-sys   = "0.1.3"
-libc          = "0.2"
-mio           = { version = "0.6", optional = true }
-tokio         = { version = "0.1", optional = true }
-tokio-io      = { version = "0.1", optional = true }
-tokio-reactor = { version = "0.1", optional = true }
+bitflags             = "1"
+futures-core-preview = { version = "=0.3.0-alpha.18", optional = true }
+inotify-sys          = "0.1.3"
+libc                 = "0.2"
+mio                  = { version = "0.6", optional = true }
+tokio-io             = { version = "=0.2.0-alpha.4", optional = true }
+tokio-net            = { version = "=0.2.0-alpha.4", optional = true, default-features = false }
```

Comment:
Some of the versions are pinned to an exact alpha version. Is there a specific reason for that?

I think requiring an exact version should usually be avoided, as that can cause problems with other crates that might have slightly different version requirements.

Reply:
These crates might have breaking changes between different alpha versions, so they are pinned to a specific version. It won't be necessary once they are stable.

Reply:
Makes sense.